pldoc/static/sass/_lib.scss

```diff
@@ -13,5 +13,3 @@
 } @else {
     @import '../../../bower_components/bi-app-sass/bi-app/bi-app-ltr';
 }
-
-@import 'syntax';
```

Comment:
NOTE: moved this file out of the Doc Site's _lib.scss partial because this lib partial is now used by the static pattern-library CSS compilation process. We don't want nor do we need this code syntax highlighting in those assets - this is needed ony for the Doc Site's code rendering currently.